components/snowflake/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/snowflake",
-  "version": "0.4.4",
+  "version": "0.4.5",
```

As per [the versioning guidelines](https://pipedream.com/docs/components/guidelines/#versioning), including a new feature (component) warrants a new `minor` version
```suggestion
  "version": "0.5.0",
```